### PR TITLE
Add "plugins" menu option in full playback view

### PR DESCRIPTION
**Pull request by Adonai**
- refactor plugin querying in a way that now any
   library activity can utilize it
- Make doc more thorough
- Hopefully fix empty plugin list bug
   (flag was added to wrong intent)

### CODE REVIEW

File: src/ch/blinkenlights/android/vanilla/LibraryActivity.java [hunk 1]
```diff
@@ -776,12 +713,7 @@ public void onClick(DialogInterface dialog, int id) {
...
 		}
 		case CTX_MENU_PLUGINS: {
-			// obtain list of plugins anew - some plugins may be installed/deleted
-			mPlugins.clear();
-			mLastRequestedCtx = intent;
-			Intent requestPlugins = new Intent(PluginUtils.ACTION_REQUEST_PLUGIN_PARAMS);
-			intent.addFlags(Intent.FLAG_INCLUDE_STOPPED_PACKAGES);
-			sendBroadcast(requestPlugins);
+			queryPluginsForSong(intent);
 			break;
```
Comment (adrian-bl) [inline src/ch/blinkenlights/android/vanilla/LibraryActivity.java]: Wouldn't it be better to call it 'queryPluginsForIntent(intent)' ?

File: src/ch/blinkenlights/android/vanilla/PlaybackActivity.java [hunk 2]
```diff
@@ -24,14 +24,16 @@
 package ch.blinkenlights.android.vanilla;
 
 import java.io.File;
-import java.util.ArrayList;
+import java.util.*;
```
Comment (adrian-bl) [inline src/ch/blinkenlights/android/vanilla/PlaybackActivity.java]: Please no wildcard imports

File: src/ch/blinkenlights/android/vanilla/PlaybackActivity.java [hunk 3]
```diff
@@ -24,14 +24,16 @@
...
 
 import android.app.Activity;
-import android.content.DialogInterface;
-import android.content.Intent;
-import android.content.SharedPreferences;
+import android.app.AlertDialog;
+import android.content.*;
```
Comment (adrian-bl) [inline src/ch/blinkenlights/android/vanilla/PlaybackActivity.java]: ditto.

File: src/ch/blinkenlights/android/vanilla/PlaybackActivity.java [hunk 4]
```diff
@@ -661,4 +685,78 @@ else if (group == CTX_MENU_GRP_FINISH)
 		return true;
 	}
 
+	/**
+	 * Sends broadcast query that wakes plugins are queries them for info.
+	 * Answers are later processed in {@link PluginBroadcastReceiver}
+	 * @param songIntent intent containing song id as {@link Long} in its "id" extra.
```
Comment (adrian-bl) [inline src/ch/blinkenlights/android/vanilla/PlaybackActivity.java]: blank line between boilerplate + @param

**Comment by Adonai** [comment]
Review comments fixed

**Pull request merged**